<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="LOGIN_stack">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>441</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="LOGIN_BT_fpass">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>340</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호 찾기</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="LOGINMAIN_id">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="inputMask">
      <string comment="  "/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>아이디</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LOGINMAIN_pass">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="inputMask">
      <string comment="  "/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>비밀번호</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LOGIN_BT_fid">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>340</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디 찾기</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="LOGIN_BT_regist">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>340</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="LOGINMAIN_title">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="LOGIN_BT_login">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="REGIST_id">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="inputMask">
      <string comment="  "/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>아이디</string>
     </property>
    </widget>
    <widget class="QLabel" name="REGIST_title">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="REGIST_pass">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="inputMask">
      <string comment="  "/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>비밀번호</string>
     </property>
    </widget>
    <widget class="QLabel" name="REGIST_idcheck">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="REGIST_pass2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="inputMask">
      <string comment="  "/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>비밀번호 확인</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="REGIST_name">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>290</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="inputMask">
      <string comment="  "/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>이름</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="REGIST_number">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>340</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="inputMask">
      <string comment="  "/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>번호</string>
     </property>
    </widget>
    <widget class="QPushButton" name="REGIST_BT_cancel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>400</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>취소</string>
     </property>
    </widget>
    <widget class="QPushButton" name="REGIST_BT_ok">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>400</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>가입하기</string>
     </property>
    </widget>
    <widget class="QLabel" name="REGIST_LB_passcheck">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>270</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="REGIST_radio">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>판매자용</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="REGIST_radio2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>일반</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="REGIST_adminnumber">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="inputMask">
      <string comment="  "/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>상호명</string>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="REGIST_BT_idcheck">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>중복확인</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LOGINMAIN_id</tabstop>
  <tabstop>LOGINMAIN_pass</tabstop>
  <tabstop>LOGIN_BT_login</tabstop>
  <tabstop>LOGIN_BT_fpass</tabstop>
  <tabstop>LOGIN_BT_fid</tabstop>
  <tabstop>LOGIN_BT_regist</tabstop>
  <tabstop>REGIST_id</tabstop>
  <tabstop>REGIST_BT_idcheck</tabstop>
  <tabstop>REGIST_pass</tabstop>
  <tabstop>REGIST_pass2</tabstop>
  <tabstop>REGIST_name</tabstop>
  <tabstop>REGIST_number</tabstop>
  <tabstop>REGIST_BT_ok</tabstop>
  <tabstop>REGIST_BT_cancel</tabstop>
  <tabstop>REGIST_radio2</tabstop>
  <tabstop>REGIST_radio</tabstop>
  <tabstop>REGIST_adminnumber</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
